depends/packages/qt.mk

```diff
@@ -178,9 +178,9 @@ define $(package)_preprocess_cmds
   patch -p1 -i $($(package)_patch_dir)/no-xlib.patch &&\
   echo "QMAKE_LINK_OBJECT_MAX = 10" >> qtbase/mkspecs/win32-g++/qmake.conf &&\
   echo "QMAKE_LINK_OBJECT_SCRIPT = object_script" >> qtbase/mkspecs/win32-g++/qmake.conf &&\
-  sed -i.old "s|QMAKE_CFLAGS            = |!host_build: QMAKE_CFLAGS            = $($(package)_cflags) $($(package)_cppflags) |" qtbase/mkspecs/win32-g++/qmake.conf && \
+  sed -i.old "s|QMAKE_CFLAGS           += |!host_build: QMAKE_CFLAGS            = $($(package)_cflags) $($(package)_cppflags) |" qtbase/mkspecs/win32-g++/qmake.conf && \
   sed -i.old "s|QMAKE_LFLAGS            = |!host_build: QMAKE_LFLAGS            = $($(package)_ldflags) |" qtbase/mkspecs/win32-g++/qmake.conf && \
```

Comment:
doesn't look like this line is working, there's no bare "QMAKE_LFLAGS" in my `win32-g++/qmake.conf` at all (qt 5.9.7)

Reply:
Digging a little into this... It would seem that `QMAKE_LFAGS` was removed in https://github.com/qt/qtbase/commit/39fc377bf105ba09e2a8f9acae467dc789b96525, and first made it into qt `v5.9.0`. We can just add `QMAKE_LFLAGS            = $($(package)_ldflags)` back in though.

Reply:
See aaec508742b89ec7b2c0a6d5a37b871c726d2598 for my fix.

Reply:
Looks good to me, thanks for addressing this